pkg/certrotation/target.go

```diff
@@ -0,0 +1,141 @@
+package certrotation
+
+import (
+	"context"
+	"crypto/x509"
+	"fmt"
+	"time"
+
+	"github.com/openshift/library-go/pkg/certs"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+
```

Comment:
nit: remove the blank line or sort the imports by their package names

Reply:
imports formated